src/java.desktop/share/classes/com/sun/java/swing/plaf/motif/MotifComboBoxUI.java

```diff
@@ -59,16 +77,6 @@ public void installUI(JComponent c) {
         arrowIcon = new MotifComboBoxArrowIcon(UIManager.getColor("controlHighlight"),
                                                UIManager.getColor("controlShadow"),
                                                UIManager.getColor("control"));
-
-        Runnable initCode = new Runnable() {
-            public void run(){
-                if ( motifGetEditor() != null ) {
-                    motifGetEditor().setBackground( UIManager.getColor( "text" ) );
-                }
-            }
-        };
-
-        SwingUtilities.invokeLater( initCode );
```

Comment:
This code was added in 1998, the intention was to postpone the initialization of the editor's background till everything in the combo box will be initialized, but:
 - Since then we have a special method configureEditor() which called every time the editor is changed and sets the correct background (see below in this file)
 - If the L&F will be changed to something else the posted event will "poison" the background of the editor by the "motif color".